docs/debugging/README.md

```diff
@@ -13,9 +13,9 @@ addition to being able to build PowerShell, you need:
 - `powershell` executable in your path (self-host if not on Windows)
 
 The .NET CLI tools *must* be on your path for Visual Studio Code.
-`Start-PSBootstrap` installs the tools to `~/.dotnet`,
+`Start-PSBootstrap` installs the tools to `~/.dotnet` (non-Windows) or `"$env:LocalAppData\Microsoft\dotnet"` (Windows),
 but does not add this to your `PATH`.
-You can do this in Bash with `export PATH=$PATH:$HOME/.dotnet`.
+You can do this in Bash with `export PATH=$PATH:$HOME/.dotnet` or in PowerShell with `$env:path = $env:path+";$env:LocalAppData\Microsoft\dotnet"`.
```

Comment:
The export command has the same behavior. I just provided the corresponding command for the Windows environment. In both cases, it should be added to the user's profile to make it permanent. PowerShell on Windows also supports direct modification of the registry key to make it system-wide, but that seemed outside the scope of this document.